assayist/processor/utils.py

```diff
@@ -132,12 +137,65 @@ def unpack_rpm(rpm_file, output_dir):
     log.info(f'Successfully unpacked {os.path.split(rpm_file)[-1]} to {output_dir}')
 
 
+def unpack_container_image(container_image_file, output_dir):
+    """
+    Unpack a container image to the specified directory.
+
+    :param str container_image_file: the path to the container image file to unpack
+    :param str output_dir: the path to unpack the container image to
+    """
+    # Unpack the manifest.json file from which we figure out the latest image layer
+    tar = tarfile.open(container_image_file)
```

Comment:
Does `tarfile.open` support the context manager syntax to ensure the tar file gets closed? Otherwise, a try/finally would work.